### PR TITLE
Implemented locationBias and locationRestriction for IOS

### PR DESCRIPTION
**Pull request by erdnj**
### Implemented locationBias and locationRestriction for IOS to solve #19 
* Also increased version numbers and updated changelog regarding to conventions.
* It is ready to release.

### CODE REVIEW

**Comment by erdnj** [comment]
@matanshukry @sidneyorlovski @ogustavobelociandt could someone review this PR ?

File: flutter_google_places_sdk/pubspec.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ dependencies:
   flutter:
     sdk: flutter
   flutter_plugin_android_lifecycle: ^2.0.2
-  flutter_google_places_sdk_ios: ^0.1.2+1
+  flutter_google_places_sdk_ios: ^0.1.2+2
```
Comment (matanshukry) [inline flutter_google_places_sdk/pubspec.yaml:14]: don't do this. I know it's annoying, but I'm tagging each published version - so we can't make a single commit upgrading the version for the main package `flutter_google_places_sdk` together with the ios package `flutter_google_places_sdk_ios` - we need 2 different commits for that.

**Review by matanshukry — CHANGES_REQUESTED** [review]

File: flutter_google_places_sdk/pubspec.yaml (hunk 1 above)
Comment (erdnj) [reply flutter_google_places_sdk/pubspec.yaml]: I have removed my edits on main package to version update as you said.

**Comment by erdnj** [comment]
@matanshukry, is this version okey to your convention ? I just bump the iOS version.

**Review by matanshukry — APPROVED** [review]

**Comment by matanshukry** [comment]
@erdnj I apologize for the delay.
I have merged the package and published a new ios version - `0.1.2+2`

**Comment by erdnj** [comment]
Better late than never. 
Thanks for the package.

**Pull request merged**